src/IntelliTect.Coalesce.CodeGeneration/Generation/Generators/RazorTemplateCSharpGenerator.cs

```diff
@@ -19,12 +20,15 @@ public RazorTemplateCSharpGenerator(RazorTemplateServices razorServices) : base(
 
         public string Namespace => GenerationContext.OutputNamespaceRoot;
 
+        [GeneratorConfig]
```

Comment:
Could you add this to the documentation? Probably in the big block of json in https://github.com/IntelliTect/Coalesce/blob/2.0/docs/pages/topics/coalesce-json.rst. Make sure to enumerate all the different generators that support this.

Another alternative would be to make the configuration also traverse the generator's inheritance hierarchy and look through configuration sections named after base classes as well if the property isn't defined on configuration for a specific generator impl. This would definitely be a whole separate feature and PR, but it could be cool to configure this for all C# generators in 3lines instead of 9.

Reply:
roger wilco

Reply:
Updated the docs with an example.